### ISSUE TITLE
dbeaver: version of the JVM is not suitable for this product

### ISSUE THREAD

**Issue by jchook** [void-linux/void-packages#34857]
### System

* xuname:  
  Void 5.15.8_1 x86_64 GenuineIntel notuptodate rrFFF
* package:  
  dbeaver-21.3.2_1

### Expected behavior

Able to launch after installing

### Actual behavior

Getting this error:

> Version 1.8.0_272 of the JVM is not suitable for this product. Version 11 or greater is required.

![image](https://user-images.githubusercontent.com/193530/148132901-04435810-a414-4e2d-a8e8-8b17117d4026.png)

### Steps to reproduce the behavior

```
sudo xbps-install -y dbeaver
dbeaver
```

**Comment by leahneukirchen** [comment]
Install openjdk11?

**Comment by jchook** [comment]
It can't seem to find the JDK. 

I had `openjdk8` installed via the `openjdk` meta package already, but uninstalling that did not help.

**Comment by jchook** [comment]
This fixes the issue:

Add the following to the *top* of `/usr/lib/dbeaver/dbeaver.ini`

```
--launcher.appendVmargs
-vm
/usr/lib/jvm/openjdk11/bin
```

**Comment by hiljusti** [comment]
Took me a while to find some steps to reproduce it, but tl;dr this would do it:

```
sudo xbps-install dbeaver
sudo xbps-install openjdk8
sudo xbps-alternatives --set openjdk8
dbeaver # fails with the error message
```

Maybe there are other ways to get to this state.

A quick fix is to do:

```
sudo xbps-alternatives --set openjdk11
```

But... I think there are reasons someone might want JDK 8 as a default, and still be able to use this app I guess. It's still an LTS version of Java, and some software still requires it for one reason or another. (Usually to run java bytecode compiled with some version <= JDK 7 that isn't supported by JDK 11)

**Comment by classabbyamp** [comment]
please test the fix I've implemented in #36653, if possible. It worked for me, hopefully it works for y'all too.
